source/clear-linux/reference/collaboration/structure-formatting.rst

```diff
@@ -0,0 +1,483 @@
+| Local navigation               |                                       | ``.. contents:: :local:``   |
+|                                |                                       | with a depth of 1           |
++--------------------------------+---------------------------------------+-----------------------------+
+| Menu selection                 |                                       | Use ``:menuselection:``     |
++--------------------------------+---------------------------------------+-----------------------------+
+| New terms                      | Italic for first use, normal for all  | ``*term*``                  |
+|                                | subsequent uses.                      |                             |
+|                                |                                       |                             |
+|                                | If it is used outside of the source   |                             |
+|                                | of definition, link the term.         |                             |
++--------------------------------+---------------------------------------+-----------------------------+
+| Product name                   | Follow correct trademark and          |                             |
+|                                | attribution guidelines.               |                             |
++--------------------------------+---------------------------------------+-----------------------------+
+| Tool names                     | Correctly capitalized, no quotes,     |                             |
+|                                | bold, or italics as the basic rule.   |                             |
+|                                |                                       |                             |
+|                                | If the tool name is the command, like |                             |
+|                                | most Linux tools, treat it like a     |                             |
+|                                | command.                              |                             |
+|                                |                                       |                             |
+|                                | If the tool name is lowercase and     |                             |
+|                                | used at the start of a sentence, use  |                             |
+|                                | bold.                                 |                             |
++--------------------------------+---------------------------------------+-----------------------------+
+
+White space and line length
+===========================
+
+Limit line length to 78 characters. The GitHub web interface forces this
+limitation for readability.
+
+Remove trailing whitespace from your documents.
+
+Code blocks and examples
+************************
+
+When providing example code or commands that are less than 10 lines, use the

```

Comment:
Is there a reason that this is limited to 10 lines? What do we do if it needs to be longer?

Reply:
Removed, viva la example code!